UPGRADE.md

```diff
@@ -1,13 +1,5 @@
 # Upgrading Steps
 
-## `v1.0.X` → `v1.1.0`
+## `v1.1.0` → `v1.1.1`
```

Comment:
Please leave the 1.0.x -> 1.1.1 upgrade steps in place and put the 1.1.0 -> 1.1.1 upgrade instructions above it